src/Assembly.js

```diff
@@ -169,8 +169,9 @@ class Assembly extends React.Component {
       }
     )
 
-    this.survey_date = DateTime.local().setLocale(this.locale).toISODate()
-    this.survey_medication_time = DateTime.local().setLocale(this.locale).toLocaleString(DateTime.TIME_24)
+    // For spanish version: Changed from .toISODate(), didn't notice any regression
+    this.survey_date = DateTime.local().setLocale(this.locale).toLocaleString()
+    this.survey_medication_time = DateTime.local().setLocale(this.locale).toLocaleString(DateTime.TIME_24_SIMPLE)
```

Comment:
Okay - this looks good.

With our next pull request, we'll clean this up into a single, global `timestamp` variable,
and `medication_recorded_time`, `symptom_recorded_time`, and `adherence_recorded_time` variables.

In the long run, it probably makes sense to group the `medication_*` variables and the `adherence_` variables together in the same way the `symtpom.*` variables are together right now.

Reply:
So `.setLocale()` actually is the setting of the zone for the String formatting of the time. It is not setting the zone. `.setZone()` is used for that.

`.local()` on the other hand is getting the current time based on the devices timezone.

Reply:
Alright.
"Locale" deals with format,
"Local" and "Zone" with time.